src/components/message-input/giphy/styles.scss

```diff
@@ -0,0 +1,54 @@
+@use '../../../shared-components/theme-engine/theme' as themeDeprecated;
+
+.giphy__container {
+  width: 500px;
+  position: absolute;
+  bottom: 65px;
+  right: 74px;
+  background-color: themeDeprecated.$background-color-tertiary;
+  padding: 5px;
+  border-radius: 5px;
+  border: 1px solid themeDeprecated.$input-arrow-color;
+
+  &--fullscreen {
+    right: 65px;
+  }
+
+  &-grid {
+    height: 440px;
+    overflow-y: overlay;
+    margin-top: 5px;
+    .loader {
```

Comment:
so just another css question here.. why use `.classname {..}` & not `&-classname` OR `__classname` (not sure)..?

Reply:
Old styles from a previous dev. I only moved the file into a new folder here, didn't take a look at the css. Something to check out on another task :)